constants.go

```diff
@@ -32,9 +35,12 @@ const DefaultCSISocket = "/run/topolvm/csi-topolvm.sock"
 // DefaultLVMdSocket is the default path of the lvmd socket file.
 const DefaultLVMdSocket = "/run/topolvm/lvmd.sock"
 
-// SizeVolConKey is the key used to obtain ephemeral inline volume size
+// EphemeralVolumeSizeKey is the key used to obtain ephemeral inline volume size
 // from the volume context
-const SizeVolConKey = "topolvm.cybozu.com/size"
+const EphemeralVolumeSizeKey = "topolvm.cybozu.com/size"
+
+// DefaultDeviceClassName is the name for the default device class.
+const DefaultDeviceClassName = "00default"
```

Comment:
`00default` should only be used as the annotation name of the default device-class, so I'd like suggest:
```suggestion
// AnnDeviceClass is the annotation name for the default device class.
const AnnDefaultDeviceClass = "00default"
```